.github/workflows/kind.yml

```diff
@@ -128,6 +129,69 @@ jobs:
         path: log.tar.gz
         retention-days: 30
 
+  test-e2e-encap-longruncases-only:
```

Comment:
@tnqn all mode irrelavent tests have been removed from other tests, I use the "modeirrelevant" to skip them in other tests and run them only in encap mode, maybe the code is not self-explanation, I will refine it and add some descriptions.

Reply:
the change you mentioned about all feature on/off, I believe it will cover more test scenarios but not likely to reduce the e2e overall time, I would like to add it with another PR.

Reply:
I feel the script is generic and shoulld run all tests by default and skip some tests only when extra parameters are provided, otherwise it's a little surprising that "--modeirrelevant" is needed to run full tests.

Reply:
Not changing all feature on/off in this PR makes sense to me, just wanted to bring up the idea here.

Reply:
done, refined the logic to use existing mode parameter "--encap-mode" and new "--time-consuming-cases-only" option to generate skip list.